test/quality/Makefile

```diff
@@ -27,26 +27,27 @@ all: capture validate ## Fetch or capture all data and run all quality checks
 
 .PHONY: validate
 validate: venv $(VULNERABILITY_LABELS)/Makefile ## Run all quality checks against already collected data
-	$(ACTIVATE_VENV) yardstick validate -r $(RESULT_SET)
+	$(YARDSTICK) validate -r $(RESULT_SET) -r $(RESULT_SET)_2022
 
 .PHONY: capture
 capture: sboms vulns ## Collect and store all syft and grype results
 
-.PHONY: capture
+.PHONY: vulns
 vulns: venv $(TEST_DB) ## Collect and store all grype results
 	$(YARDSTICK) -v result capture -r $(RESULT_SET)
+	$(YARDSTICK) -v result capture -r $(RESULT_SET)_2022
 
 $(TEST_DB):
 	@curl -o $(TEST_DB) -SsL $(TEST_DB_URL)
 
 .PHONY: sboms
 sboms: $(YARDSTICK_RESULT_DIR) venv clear-results ## Collect and store all syft results (deletes all existing results)
-	bash -c "make download-sboms || ($(YARDSTICK) -v result capture -r $(RESULT_SET) --only-producers)"
+	bash -c "make download-sboms || ($(YARDSTICK) -v result capture -r $(RESULT_SET) --only-producers && $(YARDSTICK) -v result capture -r $(RESULT_SET)_2022 --only-producers)"
 
 .PHONY: download-sboms
 download-sboms: $(VULNERABILITY_LABELS)/Makefile
 	cd vulnerability-match-labels && make venv
-	bash -c "export ORAS_CACHE=$(shell pwd)/.oras-cache && make venv && . vulnerability-match-labels/venv/bin/activate && ./vulnerability-match-labels/sboms.py download -r $(RESULT_SET)"
+	bash -c "export ORAS_CACHE=$(shell pwd)/.oras-cache && make venv && . vulnerability-match-labels/venv/bin/activate && ./vulnerability-match-labels/sboms.py download -r $(RESULT_SET) && ./vulnerability-match-labels/sboms.py download -r $(RESULT_SET)_2022"
```

Comment:
this looks like it'll pull in all SBOMs into the 2022 result set too even though they are all not needed -- not a comment for change, but something we can improve on later

Reply:
I'm missing something - AFAIK, the only SBOM in the 2022 result set is the one for the azure linux image.

Should this result set be defined elsewhere, like in vulnerability match labels?

Reply:
Based on https://github.com/anchore/vulnerability-match-labels/blob/a9a1e820e22d52c94bd70dd5bfce8f29bbdb7ce4/sboms.py#L69, it looks like this will not download extra SBOMs.